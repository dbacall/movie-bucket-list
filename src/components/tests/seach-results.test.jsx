import React from "react";
import Enzyme, { shallow, mount } from "enzyme";
import SearchResults from "../search-results";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

const mockMovies = [{ name: "Star Wars" }];

describe("SearchResults component", () => {
  it("renders", () => {
    const wrapper = shallow(<SearchResults Movies={mockMovies} />);
    expect(wrapper).toMatchSnapshot();
  });
});
