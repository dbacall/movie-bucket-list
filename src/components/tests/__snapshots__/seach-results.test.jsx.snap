// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults component renders 1`] = `ShallowWrapper {}`;
